UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API.FunctionalTests/FunctionalTests/ExchangeSetForLargeFile.cs

```diff
@@ -38,6 +35,8 @@ public async Task SetupAsync()
             ScsJwtToken = await authTokenProvider.GetScsToken();
             DataHelper = new DataHelper();
             ApiEssResponse = await ExchangeSetApiClient.GetProductIdentifiersDataAsync(DataHelper.GetProductIdentifiersForLargeMedia(), accessToken: EssJwtToken);
+            //This will allow to donwload the files
+            await Task.Delay(5000);
```

Comment:
Why a 5 second delay? Does the await GetProductIdentifiersDataAsync not complete before returning? What does the 5 second delay help with?

Reply:
Hi,

We have removed this delay and added it to the proper location in download method.

Regards,
Mayuresh Gawde